newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -65,6 +88,48 @@ export default function SummaryPage() {
               )}
             </Message>
           )}
+          {
+            <Modal
+              onClose={handleMailModalClose}
+              className={styles['user-search-modal']}
+              size="small"
+              closeIcon
+              open={mailModalOpen}
+            >
+              <Modal.Header className={styles['user-search-modal-header']}>
+                <span>E-mail participants </span>
+                <Label color="green" size="small" circular>
+                  {participantsWithEmail.length}
+                </Label>
+              </Modal.Header>
+              <Modal.Content>
+                {hasParticipantsWithoutEmail && (
+                  <>
+                    Some of your recipients do not have e-mail addresses and will not be contacted:
+                    <br />
+                    <List>
+                      {participantsWithoutEmail.map((p, index) => {
+                        return (
+                          <List.Item key={index}>
+                            <List.Icon color="red" name="close" />
+                            <List.Content>{p.name}</List.Content>
+                          </List.Item>
+                        );
+                      })}
+                    </List>
+                    <br />
+                  </>
+                )}
+                {participantsWithEmail.length} participants will be e-mailed.
```

Comment:
Should we show the actual list of participants who will be emailed there?

Reply:
As you prefer - happy to add both lists. Should we cut off the list at a certain point if it is very long?

Reply:
We could do it similar how we do it in the participant list - ie cut off after a certain number with a "show all" link.